src/Pages/messages.js

```diff
@@ -0,0 +1,8 @@
+import { defineMessages } from "react-intl";
+import { addIdAndDefaultMessageToMessages } from "Util";
+
+const messages = defineMessages({
+  Login: "Login",
+});
+
+export default addIdAndDefaultMessageToMessages(messages, 'Pages');
```

Comment:
This is example boilerplate file where default messages are defined. The values are extracted into one JSON file per locale support using the `yarn generate-messages` command.

The files are fairly simple, so I think it's okay to leave it them as JS.